plugins/aea-ledger-cosmos/setup.py

```diff
@@ -50,7 +50,7 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
-        "Framework :: AEA" "Topic :: Communications",
```

Comment:
this specifier did not work, even with fixed typo

Reply:
ok